snapcraft/models/project.py

```diff
@@ -417,10 +417,13 @@ class Project(models.Project):
     # TODO: ensure we have a test for version being retrieved using adopt-info
     # snapcraft's `version` is more general than craft-application
     version: Optional[ProjectVersion]  # type: ignore[assignment]
-    donation: Optional[Union[str, UniqueStrList]]
+    donation: Optional[UniqueStrList]  # type: ignore[assignment]
     # snapcraft's `source_code` is more general than craft-application
-    source_code: Optional[str]  # type: ignore[assignment]
-    website: Optional[str]
+    source_code: Optional[UniqueStrList]  # type: ignore[assignment]
+    license: Optional[str]  # type: ignore[assignment]
```

Comment:
Is the required?  I think this is the same as the parent model (https://github.com/canonical/craft-application/blob/c5792385aea32ef6ab811d65c51934fe2e40df80/craft_application/models/project.py#L92) and therefore can be dropped.

Reply:
Isn't `Optional[str]` and `str | none` the same thing?